src/components/Settings/SaveSearchItem.vue

```diff
@@ -109,58 +102,6 @@ export default {
 				}
 			});
 		},
-		fetchAvailableLoanCount() {
-			// extract our sortBy property
-			const sortType = this.savedSearch?.loanSearchCriteria?.sortBy ?? 'popularity';
-			// establish our new query filter object
-			const activeFilters = {};
-			// extract our saved search loan search criteria
-			const lscFilters = this.savedSearch?.loanSearchCriteria?.filters ?? {};
-			// TODO: Extend LoanSearchFilters.js methods getSearchableFilters + getInputFilters to handle a full LSC
-			// loop through each item from our saved search LSC
-			Object.keys(lscFilters).forEach(key => {
-				// exclude __typename from new filter object
-				if (key === '__typename') return false;
-				// establish our new filter value in a modifiable state
-				let filterValue = lscFilters[key];
-				// check for, + augment new filter value
-				if (filterValue !== null) {
-					// check for nested min max values
-					if (key === 'arrearsRate'
-						|| key === 'avgBorrowerCost'
-						|| key === 'defaultRate'
-						|| key === 'lenderTerm'
-						|| key === 'profitability'
-						|| key === 'riskRating'
-					) {
-						// define without including __typename
-						filterValue = getInputRange(filterValue);
-					}
-					// define final filter value
-					activeFilters[key] = filterValue;
-				}
-			});
-
-			// make our loan count query with newly formatted filter object
-			this.apollo.query({
-				query: gql`query savedSearchLoanCount(
-					$sortBy: LoanSearchSortByEnum!,
-					$filters: LoanSearchFiltersInput
-				) {
-					lend {
-						loans(sortBy: $sortBy, filters: $filters) {
-							totalCount
-						}
-					}
-				}`,
-				variables: {
-					sortBy: sortType,
-					filters: activeFilters
-				},
-			}).then(({ data }) => {
-				this.availableLoanCount = data?.lend?.loans?.totalCount ?? 0;
```

Comment:
Please remove `availableLoanCount` in a follow-up PR, as it is no longer being used.